src/constructors/regular.c

```diff
@@ -139,6 +139,155 @@ igraph_error_t igraph_star(igraph_t *graph, igraph_integer_t n, igraph_star_mode
     return IGRAPH_SUCCESS;
 }
 
+/**
+ * \ingroup generators
+ * \function igraph_wheel
+ * \brief Creates a \em wheel graph, a union of a star graph and a
+ *        cycle graph. Note that in case of a graph with 2 vertices,
+ *        the non-center vertex will have a self loop (may imagine
+ *        the earth is the center of the wheel, the moon and its orbit
+ *        is the rim of the wheel).
+ * \param graph Pointer to an uninitialized graph object, this will
+ *        be the result.
+ * \param n Integer constant, the number of vertices in the graph.
+ * \param mode Constant, gives the type of the star graph to
+ *        create. Possible values:
+ *        \clist
+ *        \cli IGRAPH_WHEEL_OUT
+ *          directed wheel graph, edges point
+ *          \em from the center to the other vertices.
+ *        \cli IGRAPH_WHEEL_IN
+ *          directed wheel graph, edges point
+ *          \em to the center from the other vertices.
+ *        \cli IGRAPH_WHEEL_MUTUAL
+ *          directed wheel graph with mutual edges.
+ *        \cli IGRAPH_WHEEL_UNDIRECTED
+ *          an undirected wheel graph is
+ *          created.
+ *        \endclist
+ * \param center Id of the vertex which will be the center of the
+ *          graph.
+ * \return Error code:
+ *         \clist
+ *         \cli IGRAPH_EINVVID
+ *           invalid number of vertices.
+ *         \cli IGRAPH_EINVAL
+ *           invalid center vertex.
+ *         \cli IGRAPH_EINVMODE
+ *           invalid mode argument.
+ *         \endclist
+ *
+ * Time complexity: O(|V|), the
+ * number of vertices in the graph.
+ *
+ * \sa \ref igraph_lattice(), \ref igraph_ring(), \ref igraph_star, \ref igraph_tree()
```

Comment:
`igraph_tree()` was recently removed and needs to be updated here:

```suggestion
 * \sa \ref igraph_lattice(), \ref igraph_ring(), \ref igraph_star, \ref igraph_kary_tree()
```

Reply:
Yes, sure that I didn't work on vector.pmt nor igraph_vector_pmt.h

Maybe I have performed a "git pull upstream develop" which caused the problem.